StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Link/LinkController.swift

```diff
@@ -176,6 +176,68 @@ import UIKit
         }
     }
 
+    /// Registers a new Link user with the provided details.
+    /// `lookupConsumer` must be called before this.
+    ///
+    /// - Parameter fullName: The full name of the user.
+    /// - Parameter phone: The phone number of the user. Expected to be in E.164 format.
+    /// - Parameter country: The country code of the user.
+    /// - Parameter completion: A closure that is called with the result of the sign up.
+    @_spi(STP) public func registerNewLinkUser(
+        fullName: String?,
+        phone: String,
+        country: String,
+        completion: @escaping (Result<Void, Error>) -> Void
+    ) {
+        guard let linkAccount else {
+            let error = IntegrationError.noActiveLinkConsumer
+            completion(.failure(error))
+            return
+        }
+        linkAccount.signUp(
+            with: phone,
+            legalName: fullName,
+            countryCode: country,
+            consentAction: .clicked_button_mobile_v1,
+            completion: { [weak self] result in
+                LinkAccountContext.shared.account = self?.linkAccount
+                completion(result)
+            }
+        )
+    }
+
+    /// Presents the Link verification flow for an existing user which requires verification.
+    ///
+    /// - Parameter viewController: The view controller from which to present the authentication flow.
+    /// - Parameter completion: A closure that is called with the result of the authentication. It returns an `AuthenticationResult` if successful, or an error if the verification failed.
```

Comment:
Same question as the `async` version, just adding a comment here for convenience in applying the docs suggestion.

```suggestion
    /// Presents the Link verification flow for an existing user which requires verification.
    /// `lookupConsumer` must be called before this.
    ///
    /// - Parameter viewController: The view controller from which to present the authentication flow.
    /// - Parameter completion: A closure that is called with the result of the authentication. It returns an `AuthenticationResult` if successful, or an error if the verification failed.
```